linkml/generators/graphqlgen.py

```diff
@@ -50,13 +53,38 @@ def visit_class_slot(self, cls: ClassDefinition, aliased_slot_name: str, slot: S
             slotrange = slotrange + "!"
         return f"\n    {lcamelcase(aliased_slot_name)}: {slotrange}"
 
+    def visit_enum(self, enum: EnumDefinition):
+        if not enum.permissible_values:
+            if self.fail_on_missing_permissible_values:
+                raise NotImplementedError(
+                    "Enumerations using `reachable_from` instead of `permissible_values` to "
+                    + "specify permissible values is not supported yet."
+                )
+            else:
+                print(
```

Comment:
Can this be a warning instead of a print? That lets it be silenced and shows up in tests

Reply:
agreed

Reply:
No more prints, using `logging.warning` for these kind of messages.